README.md

```diff
@@ -1,2 +1,22 @@
 # hw01
-Repository for first CFSS homework assignment
+Repository for first CFSS homework assignment   
+
+## HELLO MY NAME IS...   
+
+Nicholena Honors
+
+*this stuff is tough*  
+
+**but I'm getting by**
+
+### I have a ton more to learn clearly...  
+
+#For breakfast I had...
+
+* [TEA](http://g3yxi3w953w3xjjaj1xcbuay.wpengine.netdna-cdn.com/wp-content/uploads/2015/01/Dollarphotoclub_68652169-e1422257422160.jpg)
```

Comment:
This would have been cooler if you embedded the images directly in the page. For example,

```
![TEA](http://g3yxi3w953w3xjjaj1xcbuay.wpengine.netdna-cdn.com/wp-content/uploads/2015/01/Dollarphotoclub_68652169-e1422257422160.jpg)
```

Produces

![TEA](http://g3yxi3w953w3xjjaj1xcbuay.wpengine.netdna-cdn.com/wp-content/uploads/2015/01/Dollarphotoclub_68652169-e1422257422160.jpg)